### ISSUE TITLE
Docs: Fix pkg.go.dev reference

### ISSUE THREAD

**Issue by els0r** [els0r/goProbe#215]
While reading through the docs, I found a few things:

- swagger-cli is still mentioned for goProbe's API
- goProbe: the OpenAPI schema file names don't reflect the case-sensitivity of the data structures they describe
- the "reference" badge still refers to v3.0.0

**Comment by els0r** [comment]
I think we'll need to change the module path to `github.com/els0r/goProbe/v4` in order for this to play nicely.

**Comment by fako1024** [comment]
> github.com/els0r/goProbe/v4

Yes, but not only that. According to https://go.dev/doc/modules/version-numbers and https://go.dev/doc/modules/release-workflow ("Publishing breaking API changes") the steps are much more intricate:

> In the module’s go.mod file, revise the module path to append the new major version number, as in the following example:
> 
> `example.com/mymodule/v2`
> 
> Given that the module path is the module’s identifier, this change effectively creates a new module. It also changes the package path, ensuring that developers won’t unintentionally import a version that breaks their code. Instead, those wanting to upgrade will explicitly replace occurrences of the old path with the new one.
> 
> In your code, change any package paths where you’re importing packages in the module you’re updating, including packages in the module you’re updating. You need to do this because you changed your module path.

I'm also unsure about what happens to the global stuff like the README etc. I think this sentence from the documentation says it all: `A major version update – such as from v1.x.x to v2.x.x – can be a very disruptive upgrade for your module’s users. It should be a last resort.` :dizzy_face:

**Comment by els0r** [comment]
I would say we bite the bullet. And go to `/v4`. It's one monster refactor for imports, but then we're done.

Also, I don't anticipate us going to `v5` in the next 4 years, so let's get it over with and gain more of an understand just how painful.

**Comment by fako1024** [comment]
Moved ToDo "goProbe: the OpenAPI schema file names don't reflect the case-sensitivity of the data structures they describe" to new issue: #373
